packages/network-controller/src/NetworkController.ts

```diff
@@ -629,7 +629,7 @@ export class NetworkController extends BaseControllerV2<
    *
    * @returns The list of known network clients.
    */
-  getNetworkClientsById(): AutoManagedBuiltInNetworkClientRegistry &
+  getNetworkClients(): AutoManagedBuiltInNetworkClientRegistry &
```

Comment:
`getNetworkClientsById` is confusing. To me the `ById` implies you pass in an Id as an arg. cc @shanejonas @BelfordZ @mcmire